javascript/javascript1/week3/item-array-removal.js

```diff
@@ -0,0 +1,3 @@
+const names = ["Peter", "Ahmad", "Yana", "kristina", "Rasmus", "Samuel", "katrine", "Tala"];
+const nameToRemove = names.splice(1, 1);
+console.log(names);
```

Comment:
I think the assignment calls for you to use an array method to find the index.
Eg. 
```javascript 
const nameIndex = names.indexOf("Ahmad");
names.splice(nameIndex, 1);
```